lua_tests/mispec_pixbuf_2.lua

```diff
@@ -10,138 +10,133 @@ local function initBuffer(buffer, ...)
   return buffer
 end
 
-local function equalsBuffer(buffer1, buffer2)
-  return eq(buffer1:dump(), buffer2:dump())
-end
-
-
-describe('WS2812 buffers', function(it)
+describe('pixbuf tests part 2', function(it)
 
     it:should('shift LOGICAL', function()
 
-        buffer1 = ws2812.newBuffer(4, 4)
-        buffer2 = ws2812.newBuffer(4, 4)
+        buffer1 = pixbuf.newBuffer(4, pixbuf.TYPE_RGBW)
+        buffer2 = pixbuf.newBuffer(4, pixbuf.TYPE_RGBW)
 
         initBuffer(buffer1,7,8,9,12)
         initBuffer(buffer2,0,0,7,8)
         buffer1:shift(2)
-        ok(equalsBuffer(buffer1, buffer2), "shift right")
+        ok(buffer1 == buffer2, "shift right")
```

Comment:
If we use our own == her we would need to add tests for == somewhere.
Else we test the operations with untested code (our ==)

Reply:
Well, these *are* tests of our `==`.  But yes, maybe we should test for `~=` sometimes.  Here and elsewhere, more tests are welcome.

Reply:
But still we should only test one aspect in one test and not use two untested components.

Reply:
In black-box testing, it's often not possible to "only test one aspect" at a time.  Previously, this test depended on `fill` (via `initBuffer`), `shift`, and `dump`; now it depends on `fill`, `shift`, and `eq`.  I'm not opposed to having more tests, and even some that compare `==` on `dump` output, but I don't think the lack of orthogonality is new as of this diff.